app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
```

Comment:
Use ( ) while importing multiple classes/module for cleaner and more readable code.